legal-api/src/legal_api/services/naics.py

```diff
@@ -27,14 +27,18 @@ class NaicsService:
     @staticmethod
     def find_by_code(naics_code: str):
         """Return NAICS Structure matching code."""
-        naics_url = current_app.config.get('NAICS_API_URL')
-        token = AccountService.get_bearer_token()
-        response = requests.get(naics_url + '/' + naics_code, headers={
-            'Content-Type': 'application/json',
-            'Authorization': 'Bearer ' + token
-        })
-
-        if response.ok:
-            return response.json()
-        else:
-            return None
+        try:
+            naics_url = current_app.config.get('NAICS_API_URL')
```

Comment:
can we add a log statement to print out the `naics_url` here?  just want to be sure the service has the right url.